libtenzir/src/partition_transformer.cpp

```diff
@@ -493,7 +457,16 @@ auto partition_transformer(
           data.indexer_chunks.emplace_back(qf.name(), chunk);
         }
       }
-      detail::shutdown_stream_stage(self->state.stage);
+      for (auto& [_, partition_data] : self->state.data) {
+        self->request(partition_data.builder, caf::infinite, atom::persist_v)
+          .then(
+            [](resource&) {
+              // nop
+            },
+            [](caf::error&) {
+              // nop
```

Comment:
This should be handled. At least a warning, if not a hard error. We have a data loss here.